db/fixtures/miq_shortcuts.yml

```diff
@@ -10,7 +10,7 @@
   :rbac_feature_name: miq_report
   :startup: true
 - :name: miq_capacity_utilization
-  :description: Cloud Intel / Utilization
+  :description: Overview / Utilization
```

Comment:
This should be pulled out into a separate PR

Reply:
@h-kataria @bdunne @Fryguy Removed the commit with Red Hat Cloud menus, will mark up and comment the BZ as well. And renamed the PR to reflect only changes to Utilization menu.